addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -214,6 +214,10 @@ const Wysiwyg = Widget.extend({
             this._peerToPeerLoading.then(() => this.ptp.notifyAllClients('ptp_join'));
         }
 
+        this.$editable.on('click', '[data-oe-field][data-oe-sanitize-prevent-edition]', () => {
+            Dialog.alert(this, _t("Someone with escalated rights previously modified this field, you are therefore not able to modify it yourself."));
+        });
```

Comment:
I know there has been debate about this and as I said, I preferred this version but maybe "area" instead of "field", maybe a little more user friendly?

Also, I still think we should find better that an alert on every click but this PR can move on without something fancy for this IMO.